apiary.apib

```diff
@@ -1418,6 +1419,30 @@ Returns information about days off of a given user.
             + page (Pagination)
             + matches: `12` (number)
 
+### users.getWeekSchedule [POST /users.getWeekSchedule]
+
+Returns information about week schedule of a user. Only available with the *Weekly working schedule* feature.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `87982c96-f2fe-4b05-838c-ff42c0525758` (string, required)
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + data (object)
+            + periods (array)
+                + (object)
+                    + `type`: `working_hours` (enum[string])
+                        + Members
+                            + `working_hours`
+                            + `lunch_break`
+                    + `start` (object)
+                        + `day`:  `monday` (Weekday)
+                        + `time`: `09:00` (string) - The time of day in 24-hour hh:mm format. The time will be reported in the user's time zone.
```

Comment:
PP: would it be beneficial to pass that **timezone** here too?

Reply:
> PP: would it be beneficial to pass that **timezone** here too?

We specifically opted for not including it, since it's available on user info and is not really part of the week schedule.

Reply:
https://teamleader.slack.com/archives/C4XBV62C8/p1721804844287329